### PR TITLE
add typescript example

### PR DESCRIPTION
**Pull request by valentyn88**

### CODE REVIEW

File: examples/typescript/.sauce/config.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    mode: docker
+    browser: "firefox"
+    config:
+      testFiles: [ "**/*.*" ]
+  - name: "Firefox in sauce"
+    mode: sauce
+    browser: "firefox"
+    platformName: "Windows 10"
+    config:
+      testFiles: [ "**/*.*" ]
```
Comment (vrunoa) [inline examples/typescript/.sauce/config.yml:40]: @valentyn88 I dont think typescript is tied to the browser, right? only cypress itself so I'd just keep a single docker and a single sauce test.

File: examples/typescript/cypress/fixtures/example.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```
Comment (vrunoa) [inline examples/typescript/cypress/fixtures/example.json:5]: @valentyn88 are we taking this example from cypress examples? can we link back to their documentation?

File: examples/typescript/.sauce/config.yml (hunk 1 above)
Comment (valentyn88) [reply examples/typescript/.sauce/config.yml]: This issue was fixed.

File: examples/typescript/cypress/fixtures/example.json (hunk 2 above)
Comment (valentyn88) [reply examples/typescript/cypress/fixtures/example.json:5]: Yes, this example was taken from cypress. It's a JSON file and I can't add a link to the original file. We also have some other examples from their website or github repos. Maybe we can mention them in our README.md file? Do we need to have a link in they have MIT license?

Comment (vrunoa) [reply examples/typescript/cypress/fixtures/example.json:5]: still a good practice in open-source to name the upstream project were talking the examples

Comment (valentyn88) [reply examples/typescript/cypress/fixtures/example.json:5]: @vrunoa I added a note about the original repo and added typescript to the workflow.

**Review by vrunoa — APPROVED** [review]

**Pull request merged**